From: =?utf-8?q?Zbigniew_J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Sat, 12 Jan 2019 20:35:17 +0100
Subject: NEWS: document deprecation of PermissionsStartOnly= in v240

https://github.com/systemd/systemd/pull/10802#issuecomment-453772058
(cherry picked from commit 455027c98f9c8c34de171491bebc43f48376ddf5)
(cherry picked from commit 4913496e0339b4274e0756cfd35fc9085c3cd870)
---
 NEWS | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/NEWS b/NEWS
index c67b5b0..ccd2643 100644
--- a/NEWS
+++ b/NEWS
@@ -453,6 +453,14 @@ CHANGES WITH 240:
           notified about this userspace breakage quickly, but they chose to
           ignore it.
 
+        * PermissionsStartOnly= setting is deprecated (but is still supported
+          for backwards compatibility). The same functionality is provided by
+          the more flexible "+", "!", and "!!" prefixes to ExecStart= and other
+          commands.
+
+        * $DBUS_SESSION_BUS_ADDRESS environment variable is not set by
+          pam_systemd anymore.
+
         Contributions from: afg, Alan Jenkins, Aleksei Timofeyev, Alexander
         Filippov, Alexander Kurtz, Alexey Bogdanenko, Andreas Henriksson,
         Andrew Jorgensen, Anita Zhang, apnix-uk, Arkan49, Arseny Maslennikov,
